<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pairSaveWindow</class>
 <widget class="QWidget" name="pairSaveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>x-y saving</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="4">
    <widget class="QLabel" name="lbl_tick_lbl">
     <property name="text">
      <string>ticks</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QSpinBox" name="spb_ticks">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="lbl_numSaved_lbl">
     <property name="text">
      <string>values saved:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_x_lbl">
     <property name="text">
      <string>x:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lbl_frequeny_lbl">
     <property name="text">
      <string>save every</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbl_y_lbl">
     <property name="text">
      <string>y:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QComboBox" name="cob_x"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lbl_delimiter">
     <property name="text">
      <string>delimiter:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="5">
    <widget class="QComboBox" name="cob_y"/>
   </item>
   <item row="6" column="4" colspan="2">
    <widget class="QPushButton" name="btn_resetCounter">
     <property name="text">
      <string>reset counter</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QComboBox" name="cob_delimiter"/>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QLabel" name="lbl_numSaved">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QPushButton" name="btn_startStop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>start recording</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_status_lbl">
     <property name="text">
      <string>status:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="lbl_status">
     <property name="styleSheet">
      <string notr="true">color:red;</string>
     </property>
     <property name="text">
      <string>pausing</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btn_selectFile">
     <property name="text">
      <string>select file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="lbl_fileName">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;No file selected.&lt;/p&gt;&lt;p&gt;(Measure values before you can select them!)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
